.phan/stubs/wpcom-stubs.php

```diff
@@ -215,6 +215,8 @@ class Store_Product
     {
         public $product_id;
         public string $product_name;
+        public string $product_slug;
+        public string $product_type;
```

Comment:
This was the result of my PR in the other codebase. :^) Thanks for wrangling it!